core/src/main/java/org/akaza/openclinica/dao/hibernate/EventDefinitionCrfDao.java

```diff
@@ -1,46 +1,58 @@
-
-import java.util.List;
-
-import org.akaza.openclinica.domain.datamap.EventDefinitionCrf;
-
-public class EventDefinitionCrfDao extends AbstractDomainDao<EventDefinitionCrf> {
-
-    @Override
-    Class<EventDefinitionCrf> domainClass() {
-        // TODO Auto-generated method stub
-        return EventDefinitionCrf.class;
-    }
-    
-    @SuppressWarnings("unchecked")
-    public List<EventDefinitionCrf> findByStudyEventDefinitionId(int studyEventDefinitionId) {
-        String query = "from "
-                + getDomainClassName()
-                + " event_definition_crf where event_definition_crf.studyEventDefinition.studyEventDefinitionId = :studyeventdefinitionid";
-        org.hibernate.Query q = getCurrentSession().createQuery(query);
-        q.setInteger("studyeventdefinitionid", studyEventDefinitionId);
-        return (List<EventDefinitionCrf>) q.list();
-    }
-    
-    @SuppressWarnings("unchecked")
-    public List<EventDefinitionCrf> findAvailableByStudyEventDefStudy(Integer studyEventDefinitionId, Integer studyId) {
-        String query = "from " + getDomainClassName() + " do where do.studyEventDefinition.studyEventDefinitionId = :studyeventdefid " + 
-                " and do.study.studyId = :studyid and do.statusId = 1";
-        org.hibernate.Query q = getCurrentSession().createQuery(query);
-        q.setInteger("studyeventdefid", studyEventDefinitionId);
-        q.setInteger("studyid", studyId);
-        return (List<EventDefinitionCrf>) q.list();
-        
-    }
-
-    @SuppressWarnings("unchecked")
-    public List<EventDefinitionCrf> findSiteHiddenByStudyEventDefStudy(Integer studyEventDefinitionId, Integer studyId) {
-        String query = "from " + getDomainClassName() + " do where do.studyEventDefinition.studyEventDefinitionId = :studyeventdefid " + 
-                " and do.study.studyId = :studyid and do.statusId = 1 and do.hideCrf = true";
-        org.hibernate.Query q = getCurrentSession().createQuery(query);
-        q.setInteger("studyeventdefid", studyEventDefinitionId);
-        q.setInteger("studyid", studyId);
-        return (List<EventDefinitionCrf>) q.list();
-        
-    }
-}
+package org.akaza.openclinica.dao.hibernate;
+
+import java.util.List;
+
+import org.akaza.openclinica.domain.datamap.EventDefinitionCrf;
+
+public class EventDefinitionCrfDao extends AbstractDomainDao<EventDefinitionCrf> {
+
+    @Override
+    Class<EventDefinitionCrf> domainClass() {
+        // TODO Auto-generated method stub
+        return EventDefinitionCrf.class;
+    }
+
+    @SuppressWarnings("unchecked")
+    public List<EventDefinitionCrf> findByStudyEventDefinitionId(int studyEventDefinitionId) {
+        String query = "from " + getDomainClassName()
+                + " event_definition_crf where event_definition_crf.studyEventDefinition.studyEventDefinitionId = :studyeventdefinitionid";
+        org.hibernate.Query q = getCurrentSession().createQuery(query);
```

Comment:
why do we need to spell out the class path here? "org.hibernate.Query"
Can you make it to read just Query?